src/main/java/com/durov/maks/cinema/model/CinemaHall.java

```diff
@@ -7,7 +7,7 @@
 import javax.persistence.Table;
 
 @Entity
-@Table(name = "cinema_hall")
+@Table(name = "cinema_halls")
```

Comment:
```suggestion
@Table(name = "cinema-halls")
```
![image](https://user-images.githubusercontent.com/50640794/84370564-16fea580-abe1-11ea-9829-d2c60d22147b.png)

Reply:
got-it)